app/src/main/java/com/example/datarsd1/icebreaker/QuizActivity.java

```diff
@@ -0,0 +1,207 @@
+
+    private Button b1,b2,b3,b4;
+    private TextView t1;
+    private String answer;
+    private int score;
+    private CountDownTimer timer;
+    private boolean cheat;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_quiz);
+
+        answer = "";
+        score = 0;
+        cheat = getIntent().getExtras().getBoolean("CHEAT");
+
+        String[] members = {"Jessica Cherny", "Kevin Jiang", "Jared Gutierrez", "Kristin Ho", "Christine Munar", "Mudit Mittal", "Richard Hu", "Shaan Appel", "Edward Liu", "Wilbur Shi", "Young Lin", "Abhinav Koppu", "Abhishek Mangla",  "Akkshay Khoslaa", "Ally Koo", "Andy Wang", "Aneesh Jindal", "Anisha Salunkhe", "Aparna Krishnan", "Ashwin Vaidyanathan", "Cody Hsieh", "Jeffrey Zhang", "Justin Kim", "Krishnan Rajiyah", "Lisa Lee", "Peter Schafhalter", "Sahil Lamba", "Sameer Suresh", "Sirjan Kafle", "Tarun Khasnavis", "Billy Lu", "Aayush Tyagi", "Ben Goldberg", "Candice Ye", "Eliot Han", "Emaan Hariri", "Jessica Chen", "Katharine Jiang", "Kedar Thakkar", "Leon Kwak", "Mohit Katyal", "Rochelle Shen", "Sayan Paul", "Sharie Wang", "Shreya Reddy", "Shubham Goenka", "Victor Sun", "Vidya Ravikumar"};
+        Arrays.sort(members, String.CASE_INSENSITIVE_ORDER);
+        QuestionGenerator.setNames(members);
+
+        b1 = (Button) findViewById(R.id.option1);
+        b1.setOnClickListener(this);
+        b2 = (Button) findViewById(R.id.option2);
+        b2.setOnClickListener(this);
+        b3 = (Button) findViewById(R.id.option3);
+        b3.setOnClickListener(this);
+        b4 = (Button) findViewById(R.id.option4);
+        b4.setOnClickListener(this);
+
+        ImageButton exitButton = (ImageButton) findViewById(R.id.returnButton);
+        exitButton.setOnClickListener(this);
+
+        t1 = (TextView) findViewById(R.id.scoreView);
+        t1.setText("Score: "+score);
+
+        final TextView timerText = (TextView) findViewById(R.id.countdown);
+        timer = new CountDownTimer(6000,1000) {
+            @Override
+            public void onTick(long millisUntilFinished)
+            {
+                timerText.setText("Seconds Remaining: " + millisUntilFinished / 1000);
+            }
+
+            @Override
+            public void onFinish()
+            {
+                Toast.makeText(getApplicationContext(), "The correct answer was " + answer, Toast.LENGTH_LONG).show();
+                askQuestion();
+            }
+        };
+
+        askQuestion();
+    }
+
+    protected void askQuestion()
+    {
+        t1.setText("Score: "+score);
+        Question q = QuestionGenerator.makeQuestion();
+
+        b1.setText(q.getOptions()[0]);
+        b2.setText(q.getOptions()[1]);
+        b3.setText(q.getOptions()[2]);
+        b4.setText(q.getOptions()[3]);
+        answer = q.getName();
+
+        if (cheat)
```

Comment:
If you're going to call something four times, it's better to make it a private method.